tests/debugger/test_debugger.py

```diff
@@ -155,19 +183,37 @@ def wait_for_probe(self, data):
 @scenarios.debugger_method_probes_snapshot
 class Test_Debugger_Method_Probe_Snaphots(_Base_Debugger_Snapshot_Test):
     log_probe_response = None
+    metric_probe_response = None
+    span_probe_response = None
+    span_decoration_probe_response = None
 
     def setup_method_probe_snaphots(self):
         interfaces.library.wait_for(self.wait_for_remote_config, timeout=30)
         interfaces.agent.wait_for(self.wait_for_probe, timeout=30)
         self.log_probe_response = weblog.get("/debugger/log")
+        self.metric_probe_response = weblog.get("/debugger/metric/1")
+        self.span_probe_response = weblog.get("/debugger/span")
+        self.span_decoration_probe_response = weblog.get("/debugger/span-decoration/asd/1")
 
     def test_method_probe_snaphots(self):
         assert self.remote_config_is_sent is True
         assert self.probe_installed is True
-        assert self.log_probe_response.status_code == 200
 
-        expected_data = ["logProbe-installed"]
-        validate_data(expected_data, expected_data)
+        assert self.log_probe_response.status_code is 200
```

Comment:
Even if it works, `is` must be reserved for reference comparizon. See the output form the interpreter : 

![image](https://github.com/DataDog/system-tests/assets/11915659/c703832f-27d3-4f64-977b-3ac64dc196b5)


```suggestion
        assert self.log_probe_response.status_code == 200
```

Reply:
Fixed here: de6a537028ac016bb10d972b0903b79c9c02bf26

Reply:
I changed back to `is True` here a04a833aa5bd8caaf3c415187c00b2f6648cdf1d